Microservices/TemplateProject/pom.xml

```diff
@@ -4,10 +4,9 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<parent>
-		<groupId>cs4337.group-8</groupId>
-		<artifactId>parent-microservices</artifactId>
-		<version>0.0.1-SNAPSHOT</version>
-		<relativePath>../pom.xml</relativePath>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.3.4</version>
```

Comment:
This isnt the right pom.xml to remove this project as it is still referenced in the parent one.

Reply:
referenced here: https://github.com/Third-Floor-CSIS/cs4337-Big-Data-Group/blob/c0d507d1bfd13bad15b762140991ed33fe4e1d6d/Microservices/pom.xml#L19

Reply:
No this doesnt reference the parent. It references the Spring boot parent which is the version of Spring boot to derive from. This exists in the parent because each of them are Spring boot applications, however this is now a standalone spring project. 

See how the other projects derive from parent in contrast:
```xml
	<parent>
        <groupId>cs4337.group-8</groupId>
        <artifactId>parent-microservices</artifactId>
        <version>0.0.1-SNAPSHOT</version>
        <relativePath>../pom.xml</relativePath>
</parent>
```

Vs this:
```xml
<parent>
            <groupId>org.springframework.boot</groupId>
            <artifactId>spring-boot-starter-parent</artifactId>
            <version>3.3.4</version>
	</parent>
```

Reply:
> No this doesnt reference the parent.

I never said it referenced the parent, I said that its referenced *in* the parent here: https://github.com/Third-Floor-CSIS/cs4337-Big-Data-Group/blob/c0d507d1bfd13bad15b762140991ed33fe4e1d6d/Microservices/pom.xml#L19

Reply:
Oh right, good catch